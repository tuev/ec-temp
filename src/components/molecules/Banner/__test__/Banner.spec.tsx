/**
 *
 * Tests for Banner
 *
 */

import React from 'react'

import { render, fireEvent } from '@testing-library/react'
// import Banner from '..'

describe('App Banner', () => {
  it('Test get defaasdasdkjghaskjd ', () => {
    expect(true).toBeTruthy()
  })
})
