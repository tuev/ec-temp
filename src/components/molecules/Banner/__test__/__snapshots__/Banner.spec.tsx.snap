// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Banner Test get default Banner no params  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c1 {
  padding-bottom: 1rem;
  max-width: 50%;
}

.c3 {
  padding-bottom: 2rem;
  max-width: 50%;
}

.c2 {
  font-size: 48px !important;
  line-height: 1.5 !important;
  font-family: 'Playfair Display' !important;
  font-weight: bold !important;
  color: #fff;
}

.c4 {
  font-size: 24px !important;
  line-height: 1.5 !important;
  font-family: 'Montserrat Alternates' !important;
  font-weight: bold !important;
  color: #fff;
}

.c5 {
  min-width: 163px !important;
  min-height: 48px !important;
  font-size: 14px;
  border-radius: 32px;
}

.c0 {
  height: calc(100vh - 80px);
  width: 100vw;
  background-image: url(mock.png);
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center;
  padding: 64px 128px;
}

<body>
    <div>
      <div
        class="c0"
      >
        <div
          class="c1"
        >
          <p
            class="MuiTypography-root c2 MuiTypography-body1"
            customcolor="white"
            customvariant="header1"
          >
            banner title
          </p>
        </div>
        <div
          class="c3"
        >
          <p
            class="MuiTypography-root c4 MuiTypography-body1"
            customcolor="white"
            customvariant="header2"
          />
        </div>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined c5 MuiButton-outlinedPrimary"
          data-testid="banner-btn"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Shop now
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-AykKF laifGC"
    >
      <div
        class="sc-AykKC hHpGgs"
      >
        <p
          class="MuiTypography-root sc-AykKD iLGAmO MuiTypography-body1"
          customcolor="white"
          customvariant="header1"
        >
          banner title
        </p>
      </div>
      <div
        class="sc-AykKC gXLXEJ"
      >
        <p
          class="MuiTypography-root sc-AykKD gddKlQ MuiTypography-body1"
          customcolor="white"
          customvariant="header2"
        />
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined sc-AykKE kVxlLt MuiButton-outlinedPrimary"
        data-testid="banner-btn"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Shop now
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`App Banner Test get default Banner no params with action 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    .c1 {
  padding-bottom: 1rem;
  max-width: 50%;
}

.c3 {
  padding-bottom: 2rem;
  max-width: 50%;
}

.c2 {
  font-size: 48px !important;
  line-height: 1.5 !important;
  font-family: 'Playfair Display' !important;
  font-weight: bold !important;
  color: #fff;
}

.c4 {
  font-size: 24px !important;
  line-height: 1.5 !important;
  font-family: 'Montserrat Alternates' !important;
  font-weight: bold !important;
  color: #fff;
}

.c5 {
  min-width: 163px !important;
  min-height: 48px !important;
  font-size: 14px;
  border-radius: 32px;
}

.c0 {
  height: calc(100vh - 80px);
  width: 100vw;
  background-image: url(mock.png);
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center;
  padding: 64px 128px;
}

<div>
      <div
        class="c0"
      >
        <div
          class="c1"
        >
          <p
            class="MuiTypography-root c2 MuiTypography-body1"
            customcolor="white"
            customvariant="header1"
          >
            banner title
          </p>
        </div>
        <div
          class="c3"
        >
          <p
            class="MuiTypography-root c4 MuiTypography-body1"
            customcolor="white"
            customvariant="header2"
          />
        </div>
        <button
          class="MuiButtonBase-root MuiButton-root MuiButton-outlined c5 MuiButton-outlinedPrimary"
          data-testid="banner-btn"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-label"
          >
            Shop now
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-AykKF laifGC"
    >
      <div
        class="sc-AykKC hHpGgs"
      >
        <p
          class="MuiTypography-root sc-AykKD iLGAmO MuiTypography-body1"
          customcolor="white"
          customvariant="header1"
        >
          banner title
        </p>
      </div>
      <div
        class="sc-AykKC gXLXEJ"
      >
        <p
          class="MuiTypography-root sc-AykKD gddKlQ MuiTypography-body1"
          customcolor="white"
          customvariant="header2"
        />
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-outlined sc-AykKE kVxlLt MuiButton-outlinedPrimary"
        data-testid="banner-btn"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          Shop now
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
